packages/twenty-front/src/modules/object-record/record-group/utils/sortedInsert.ts

```diff
@@ -0,0 +1,20 @@
+export const sortedInsert = <T>(
```

Comment:
I'm not a big fan of this function which is doing two things: sorting + insert. Not sure we need a combined logic here.
why not just sorting where needed and inserting when needed?

Reply:
ok, let's give it a local naming then: recordGroupSortIndex so people don't start using it